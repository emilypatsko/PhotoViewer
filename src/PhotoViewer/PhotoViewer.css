.main-photo-view {
    display: flex;
    flex-direction: column;
    align-items: center;
}

.main-photo-view img {
    width: 600px;
}

h1 {
    color: blue;
}