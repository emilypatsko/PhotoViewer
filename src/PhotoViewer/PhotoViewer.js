import React from "react"; // import React for access to JSX
import './PhotoViewer.css';

function PhotoViewer() {
    return (
        <div className = "main-photo-view">
            Selected photo:
            <img src = "https://picsum.photos/300/200"></img>
        </div>
    );
}

export {PhotoViewer};