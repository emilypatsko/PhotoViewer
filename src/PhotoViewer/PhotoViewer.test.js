import { PhotoViewer } from './PhotoViewer.js';
// import { renderer } from '@testing-library/react';
// const renderer = require('react-test-renderer');
import React from 'react';
import renderer from 'react-test-renderer';

describe("PhotoViewer", () => {
    it("should match snapshot with src prop", async () => {
        const tree = renderer.create(<PhotoViewer src="https://picsum.photos/id/879/600/400"/>).toJSON();

        expect(tree).toMatchSnapshot();
    })
})
