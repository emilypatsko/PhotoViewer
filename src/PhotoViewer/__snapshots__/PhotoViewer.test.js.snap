// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PhotoViewer should match snapshot with src prop 1`] = `
<div
  className="main-photo-view"
>
  Selected photo:
  <img
    src="https://picsum.photos/id/879/600/400"
  />
</div>
`;
